Source/Engine/Core/Math/BoundingBox.cpp

```diff
@@ -14,6 +15,56 @@ String BoundingBox::ToString() const
     return String::Format(TEXT("{}"), *this);
 }
 
+ bool BoundingBox::IsOnView(Camera* camera)
+{
+     if (&camera == nullptr)
```

Comment:
I'd probably also change this to `if (camera)` to be consistent with the rest of the engine